public/js/src/client.js

```diff
@@ -540,6 +557,14 @@ $(document).ready(function(){
 	function PROCESS ( args ) {
 		args = lowBandwidthDecode(args);
 		var commandType = args["commandType"];
+/* 
+	Rather than an if statement for each of these, why not pass an index of 
+	OPERATIONS as the args["commandType"] and access the desired method with 
+	the following?
+		OPERATIONS[commandType](args)
+	You'd need to do the parsing of args in each of OPERATIONS' methods, but
+	you'd save up to 8 operations.
+*/
```

Comment:
I actually thought about this, and I wanted to be explicit in my conventions.  I liked the idea that each of my command would have an arbitrary code which could be modified without impacting the semantics of this section.  This is also way easier to read...